src/univariates.jl

```diff
@@ -465,20 +465,20 @@ for fun in [:pdf, :logpdf,
 
     @eval begin
         function ($_fun!)(r::AbstractArray, d::UnivariateDistribution, X::AbstractArray)
-            for i in 1 : length(X)
+            for i in linearindices(X)
```

Comment:
I see. However, I still think `eachindex` will be slightly better in this case. In most cases, it will return the same as `linearindices` but in a few cases it will return something better.